apps/condo/components/TicketStatusSelect.tsx

```diff
@@ -0,0 +1,85 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import React, { useCallback, useMemo } from 'react'
+import { Select } from 'antd'
+import styled from '@emotion/styled'
+import { useIntl } from '@core/next/intl'
+import { green , grey, orange, red, yellow } from '@ant-design/colors'
+import { Ticket , TicketStatus } from '../utils/clientSchema/Ticket'
+import { runMutation } from '../utils/mutations.utils'
+import { CANCEL, COMPLETE, DEFFER, IN_PROGRESS, OPEN } from '../constants/statusIds'
+import { colors } from '../constants/style'
+import { getTicketLabel } from '../utils/ticket'
+
+const statusSelectColors = {
+    [OPEN]: {
+        color: yellow[9],
```

Comment:
I think it's part of `TicketStatus` and we don't need to extract it inside the code. And we also want to support different languages and the constants for different languages are different.